clang/test/CheckedCRewriter/anonstruct.c

```diff
@@ -1,15 +1,18 @@
-// RUN: cconv-standalone -alltypes %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_ALL","CHECK" %s
-// RUN: cconv-standalone %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_NOALL","CHECK" %s
-// RUN: cconv-standalone %s -- | %clang -c -fcheckedc-extension -x c -o /dev/null -
+// RUN: cconv-standalone -alltypes -addcr %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_ALL","CHECK" %s
+// RUN: cconv-standalone -addcr %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_NOALL","CHECK" %s
+// RUN: cconv-standalone -addcr %s -- | %clang -c -fcheckedc-extension -x c -o /dev/null -
+// RUN: cconv-standalone -output-postfix=checked -alltypes %s
+// RUN: cconv-standalone -alltypes %S/anonstruct.checked.c -- | count 0
+// RUN: rm %S/anonstruct.checked.c
 
 struct { 
     int *data; 
+	//CHECK: _Ptr<int> data; 
 } *x; 
-//CHECK: _Ptr<int> data;
-//CHECK: } *x;
 
 /*ensure trivial conversion*/
```

Comment:
Why not change this test to actually use the anonymous struct in some way? Seems a wasted opportunity ...

Reply:
I agree. This was one I auto-generated annotations for as part of an experiment, but even some ones I did manually struck me as rather outdated because I already saw code like it before. This one has its place and I can spruce it up, but I'll compile a list of tests that I think have been subsumed by other tests. (Some autogend tests fall in this category too unfortunately).